### DISCUSSION TITLE
Starting out with Vue and PocketBase

### DISCUSSION THREAD

**Discussion by Shidoengie** [pocketbase/pocketbase#2442] [Q&A]
hey im starting out in webdev and i decided to use pocketbase and vue.
thing is i dont know what im doing, and i need help hehe
Im mostly having issues with the server stuff and how integrate pocketbase with vue
im also open to changes in tooling
oh and btw is there a discord server???

**Comment by dschissler** [comment]
>  Im mostly having issues with the server stuff and how integrate pocketbase with vue

In the data directory (`pb_data` by default) you can make a `public` directory.  Then put `index` there.  Try to get that loading static first.
